From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Thu, 29 Aug 2019 14:04:15 -0700
Subject: ANDROID: gki_defconfig: enable CONFIG_QCOM_{COMMAND_DB,RPMH,PDC}

CONFIG_ARCH_QCOM is a dependency of the above and selects
CONFIG_{PINCTRL, REGULATOR, TMPFS}.

Bug: 133441279
Bug: 133441092
Bug: 133440650
Change-Id: I22c37946ec3a62ccbd3fa65bbc09076964d86475
Signed-off-by: Tri Vo <trong@google.com>
---
 arch/arm64/configs/gki_defconfig | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 604b97841737..6aae6d55e96c 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -36,6 +36,7 @@ CONFIG_EMBEDDED=y
 CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
 CONFIG_PROFILING=y
+CONFIG_ARCH_QCOM=y
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=32
 CONFIG_SECCOMP=y
@@ -279,7 +280,6 @@ CONFIG_HW_RANDOM_VIRTIO=y
 # CONFIG_I2C_HELPER_AUTO is not set
 CONFIG_SPI=y
 CONFIG_SPMI=y
-CONFIG_PINCTRL=y
 CONFIG_PINCTRL_AMD=y
 CONFIG_POWER_AVS=y
 # CONFIG_HWMON is not set
@@ -290,7 +290,6 @@ CONFIG_DEVFREQ_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_MFD_ACT8945A=y
 CONFIG_MFD_SYSCON=y
-CONFIG_REGULATOR=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
@@ -357,12 +356,15 @@ CONFIG_COMMON_CLK_SCPI=y
 CONFIG_HWSPINLOCK=y
 CONFIG_MAILBOX=y
 CONFIG_ARM_SMMU=y
+CONFIG_QCOM_COMMAND_DB=y
+CONFIG_QCOM_RPMH=y
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
 CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_EXTCON=y
 CONFIG_PWM=y
+CONFIG_QCOM_PDC=y
 CONFIG_GENERIC_PHY=y
 CONFIG_RAS=y
 CONFIG_ANDROID=y
@@ -380,7 +382,6 @@ CONFIG_FUSE_FS=y
 CONFIG_OVERLAY_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
-CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 # CONFIG_EFIVAR_FS is not set
 CONFIG_SDCARD_FS=y
